content/02.introduction.md

```diff
@@ -121,6 +121,16 @@ Pathogenesis is made up of three major components: entry, replication, and sprea
 #### Immunological Mechanisms of Coronavirus-driven Disease in Humans
 
 Coronaviruses are known to cause respiratory illnesses in humans through the following possible mechanisms...[Summarize relevant mechanisms for cell entry & address evidence for/against ACE2 being important, etc.]
+One way that humans respond to viral threats is through the immune response.
+The human immune system utilizes a variety of innate and adaptive responses to protect against the pathogens it encounters.
+The innate immune system consists of barriers such as the skin, mucous secretions, neutrophils, macrophages, and dendritic cells.
+It also includes cell-surface receptors that can recognize the molecular patterns of pathogens.
+The adaptive immune system utilizes antigen-specific receptors that are expressed on B and T lymphocytes.
+ These components of the immune system typically act together; the innate response acts first, and the adaptive response begins to act several days after initial infection following the clonal expansion of T and B cells [@url:https://pubmed.ncbi.nlm.nih.gov/16455341 ].
+
+After SARS-CoV-2 enters into a host cell, its antigen is presented by major histocompatibility complex 1 (MHC 1) molecules and is then recognized by cytotoxic T lymphocytes.
```

Comment:
```suggestion
After a virus enters into a host cell, its antigen is presented by major histocompatibility complex 1 (MHC 1) molecules and is then recognized by cytotoxic T lymphocytes.
```
I think this is just broadly the case, not SARS-CoV-2 specific, right?

Reply:
yes, @rando2, this is broadly the case :+1: